services/datamanager/builtin/builtin.go

```diff
@@ -718,7 +719,10 @@ func generateMetadataKey(component, method string) string {
 }
 
 func pollFilesystem(ctx context.Context, wg *sync.WaitGroup, captureDir string, syncer datasync.Manager, logger logging.Logger) {
-	// t := time.NewTicker(filesystemPollInterval)
+	if runtime.GOOS == "android" {
+		logger.Warn("File deletion if disk is full is not currently supported on Android")
```

Comment:
Debug log? And check to make sure that we don't hit this on a macbook or pi.

Reply:
Already tested on both, will update to debug.